centreon/src/App/MonitoringConfiguration/Infrastructure/Dbal/DbalConnectorRepository.php

```diff
@@ -61,6 +69,9 @@ public function findById(ConnectorId $id): ?Connector
         $qb->select(
             'id',
             'name',
+            'command_line',
+            'description',
+            'enabled',
```

Comment:
```suggestion
$qb->select(...self::getSelectColumns())
```

how about we use the getSelectColumns, to be consistent with the other methods ?